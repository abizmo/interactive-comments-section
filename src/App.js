function App() {
  return (
    <h1>Interactive comments section</h1>
  );
}

export default App;
