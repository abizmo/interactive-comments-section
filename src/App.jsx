import React from 'react';

import Comments from './containers/Comments';
import Theme from './theme';

function App() {
  return (
    <Theme>
      <Comments />
    </Theme>
  );
}

export default App;
