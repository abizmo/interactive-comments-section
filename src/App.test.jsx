import React from 'react';
import { render, screen } from '@testing-library/react';
import App from './App';

test('renders Interactive comments section', () => {
  render(<App />);
  const linkElement = screen.getByText(/Interactive comments section/i);
  expect(linkElement).toBeInTheDocument();
});
