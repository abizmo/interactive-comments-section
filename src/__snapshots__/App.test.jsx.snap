// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Interactive comments section 1`] = `
<div>
  <div
    class="sc-jqUVSM cARRwl"
  >
    <article
      class="sc-eCYdqJ eopLEu"
    >
      <header>
        <picture
          style="width: 2rem;"
        >
          <source
            srcset="/./images/avatars/image-amyrobson.webp"
            type="image/webp"
          />
          <img
            alt="amyrobson"
            src="/./images/avatars/image-amyrobson.png"
            style="width: 100%;"
          />
        </picture>
        <span
          class="sc-gKXOVf cAhEkI"
        >
          amyrobson
        </span>
        <span>
          1 month ago
        </span>
      </header>
      <p>
        Impressive! Though it seems the drag feature could be improved. But overall it looks incredible. You've nailed the design and the responsiveness at various breakpoints works really well.
      </p>
      <div
        id="votes"
      >
        <div
          class="sc-dkzDqf fIRSEx"
          role="widget"
        >
          <button
            class="sc-hKMtZM dbWWXw"
            type="button"
          >
            <svg>
              plus.svg
            </svg>
            <span
              class="sr-only"
            >
              Plus
            </span>
          </button>
          <p>
            12
          </p>
          <button
            class="sc-hKMtZM dbWWXw"
            type="button"
          >
            <svg>
              minus.svg
            </svg>
            <span
              class="sr-only"
            >
              Minus
            </span>
          </button>
        </div>
      </div>
      <div
        id="buttons"
      >
        <button
          class="sc-bczRLJ gbCjDT"
          color="primary"
          type="button"
        >
          <svg>
            reply.svg
          </svg>
          Reply
        </button>
      </div>
    </article>
    <article
      class="sc-eCYdqJ eopLEu"
    >
      <header>
        <picture
          style="width: 2rem;"
        >
          <source
            srcset="/./images/avatars/image-maxblagun.webp"
            type="image/webp"
          />
          <img
            alt="maxblagun"
            src="/./images/avatars/image-maxblagun.png"
            style="width: 100%;"
          />
        </picture>
        <span
          class="sc-gKXOVf cAhEkI"
        >
          maxblagun
        </span>
        <span>
          2 weeks ago
        </span>
      </header>
      <p>
        Woah, your project looks awesome! How long have you been coding for? I'm still new, but think I want to dive into React as well soon. Perhaps you can give me an insight on where I can learn React? Thanks!
      </p>
      <div
        id="votes"
      >
        <div
          class="sc-dkzDqf fIRSEx"
          role="widget"
        >
          <button
            class="sc-hKMtZM dbWWXw"
            type="button"
          >
            <svg>
              plus.svg
            </svg>
            <span
              class="sr-only"
            >
              Plus
            </span>
          </button>
          <p>
            5
          </p>
          <button
            class="sc-hKMtZM dbWWXw"
            type="button"
          >
            <svg>
              minus.svg
            </svg>
            <span
              class="sr-only"
            >
              Minus
            </span>
          </button>
        </div>
      </div>
      <div
        id="buttons"
      >
        <button
          class="sc-bczRLJ gbCjDT"
          color="primary"
          type="button"
        >
          <svg>
            reply.svg
          </svg>
          Reply
        </button>
      </div>
    </article>
    <div
      class="sc-jqUVSM sc-kDDrLX cARRwl cJWLJF"
    >
      <article
        class="sc-eCYdqJ eopLEu"
      >
        <header>
          <picture
            style="width: 2rem;"
          >
            <source
              srcset="/./images/avatars/image-ramsesmiron.webp"
              type="image/webp"
            />
            <img
              alt="ramsesmiron"
              src="/./images/avatars/image-ramsesmiron.png"
              style="width: 100%;"
            />
          </picture>
          <span
            class="sc-gKXOVf cAhEkI"
          >
            ramsesmiron
          </span>
          <span>
            1 week ago
          </span>
        </header>
        <p>
          <span
            class="sc-jSMfEi iqwptk"
          >
            @
            maxblagun
             
          </span>
          If you're still new, I'd recommend focusing on the fundamentals of HTML, CSS, and JS before considering React. It's very tempting to jump ahead but lay a solid foundation first.
        </p>
        <div
          id="votes"
        >
          <div
            class="sc-dkzDqf fIRSEx"
            role="widget"
          >
            <button
              class="sc-hKMtZM dbWWXw"
              type="button"
            >
              <svg>
                plus.svg
              </svg>
              <span
                class="sr-only"
              >
                Plus
              </span>
            </button>
            <p>
              4
            </p>
            <button
              class="sc-hKMtZM dbWWXw"
              type="button"
            >
              <svg>
                minus.svg
              </svg>
              <span
                class="sr-only"
              >
                Minus
              </span>
            </button>
          </div>
        </div>
        <div
          id="buttons"
        >
          <button
            class="sc-bczRLJ gbCjDT"
            color="primary"
            type="button"
          >
            <svg>
              reply.svg
            </svg>
            Reply
          </button>
        </div>
      </article>
      <article
        class="sc-eCYdqJ eopLEu"
      >
        <header>
          <picture
            style="width: 2rem;"
          >
            <source
              srcset="/./images/avatars/image-juliusomo.webp"
              type="image/webp"
            />
            <img
              alt="juliusomo"
              src="/./images/avatars/image-juliusomo.png"
              style="width: 100%;"
            />
          </picture>
          <span
            class="sc-gKXOVf cAhEkI"
          >
            juliusomo
          </span>
          <span
            class="sc-iBkjds iXTMIL"
          >
            you
          </span>
          <span>
            2 days ago
          </span>
        </header>
        <p>
          <span
            class="sc-jSMfEi iqwptk"
          >
            @
            ramsesmiron
             
          </span>
          I couldn't agree more with this. Everything moves so fast and it always seems like everyone knows the newest library/framework. But the fundamentals are what stay constant.
        </p>
        <div
          id="votes"
        >
          <div
            class="sc-dkzDqf fIRSEx"
            role="widget"
          >
            <button
              class="sc-hKMtZM dbWWXw"
              type="button"
            >
              <svg>
                plus.svg
              </svg>
              <span
                class="sr-only"
              >
                Plus
              </span>
            </button>
            <p>
              2
            </p>
            <button
              class="sc-hKMtZM dbWWXw"
              type="button"
            >
              <svg>
                minus.svg
              </svg>
              <span
                class="sr-only"
              >
                Minus
              </span>
            </button>
          </div>
        </div>
        <div
          id="buttons"
        >
          <button
            class="sc-bczRLJ bDIsdp"
            color="secondary"
            type="button"
          >
            <svg>
              delete.svg
            </svg>
            Delete
          </button>
          <button
            class="sc-bczRLJ gbCjDT"
            color="primary"
            type="button"
          >
            <svg>
              edit.svg
            </svg>
            Edit
          </button>
        </div>
      </article>
    </div>
    <form
      class="sc-ftvSup bhApFN"
    >
      <picture
        style="width: 2rem;"
      >
        <source
          srcset="/./images/avatars/image-juliusomo.webp"
          type="image/webp"
        />
        <img
          alt="juliusomo"
          src="/./images/avatars/image-juliusomo.png"
          style="width: 100%;"
        />
      </picture>
      <textarea
        aria-label="New comment"
        class="sc-papXJ hmxrRz"
        name="new-comment"
        placeholder="Add comment..."
        type="text"
      />
      <button
        class="sc-bczRLJ sc-gsnTZi gbCjDT jNxzwF"
        color="primary"
        type="submit"
      >
        send
      </button>
    </form>
  </div>
</div>
`;
